sandbox-templates/rehype-debug/src/index.js

```diff
@@ -0,0 +1,26 @@
+// ⚠️ Important! Please make sure the dependencies are up to date.
+// On code sandbox, you can refresh them in the Dependencies section (left-bottom)
+// On stackblitz, you can open the package.json file, update the versions,
+// then run npm install in the stackblitz terminal
+
+import { rehype } from "rehype";
+
+const sourceHtml = `
+<html>
+  <head></head>
+  <body>
+    <h1>title</h1>
+    <p>lorem ipsum</p>
+  </body>
+</html>
+`;
```

Comment:
This document is missing a doctype and a title element for it to be a conforming HTML document.